thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/dao/TestGenericPojoDao.java

```diff
@@ -148,19 +152,15 @@ public void filterWithOffsetWithoutLimitTest() {
     final DaoFilter filter = new DaoFilter()
         .setOffset(offset)
         .setBeanClass(AnomalyDTO.class);
-    final List<AnomalyDTO> anomalies = dao.filter(filter);
-    assertThat(anomalies).isNotNull();
-    assertThat(anomalies).isEmpty();
+    assertThatThrownBy(() -> dao.filter(filter)).hasMessage(ERR_OFFSET_WITHOUT_LIMIT.getMessage());
```

Comment:
nice! just update this to `IllegalArgumentException` as per above comment. We'll use standard exceptions in the backend and use ThirdEyeException to communicate that automatically to the UI.